---
bool success
string error_message
cob_articulation_controller/ModelMsg model
